lib/jekyll/static_file.rb

```diff
@@ -11,6 +8,7 @@ class StaticFile
     # base - The String path to the <source>.
     # dir  - The String path between <source> and the file.
     # name - The String filename of the file.
+    # rubocop: disable ParameterLists
```

Comment:
I'm not sure about this disabling...
It seems like second argument - base - is always taken from site.source variable:
https://github.com/jekyll/jekyll/blob/master/lib/jekyll/readers/static_file_reader.rb#L18
https://github.com/jekyll/jekyll/blob/master/lib/jekyll/collection.rb#L70

Do you think we should change the signature and usage of this class? Or just ignore it?